docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md

```diff
@@ -0,0 +1,71 @@
+# Thingy:91
+
+Integrate your Nordic Thingy:91 board.
+
+## Prerequisites
+
+- The Thingy:91 board with a USB cable.
+- Installed **minicom** (for Linux) or RealTerm (for Windows) or other serial communication program.
+- A user with access to the Coiote IoT Device Management platform and appropriate permissions.
+
+## Step 1: Getting Zephyr and Python dependencies
+
+To get the Zephyr SDK and dependencies follow the first 4 steps of the instruction provided by [the Zephyr Project](https://docs.zephyrproject.org/latest/getting_started/index.html).
+
+## Step 2: Cloning the Anjay zephyr repository
+
+Enter the command line interface on your machine, then paste and run the following command:
+
+   ```
+   git clone https://github.com/AVSystem/Anjay-zephyr-client
+   ```
+
+## Step 3: Compiling the board
+
+0. Connect the Thingy:91 board to a USB port of your machine.
+0. Set West manifest path to `Anjay-zephyr-client/demo`, manifest file to `west-nrf.yml`, and do `west update`:
+
+    ```
+    west config manifest.path Anjay-zephyr-client/demo
+    west config manifest.file west-nrf.yml
+    west update
+    ```
+
+0. Compile the project for **Thingy:91** using `west build -b thingy91_nrf9160ns` in the demo directory.
```

Comment:
After compiling you need to `program the board using external programmer` in case of Thingy:91